src/jimgw/core/population/injection_util.py

```diff
@@ -0,0 +1,88 @@
+"""
+TODO: This file should be refactored into better structured modules.
+Right now this is here just because I am developing the injection pipeline
+"""
+
+import numpy as np
+import jax
+from jimgw.core.prior import (
+    UniformPrior,
+    UniformSpherePrior,
+    SinePrior,
+    CosinePrior,
+    PowerLawPrior,
+    CombinePrior,
+)
+from jimgw.core.single_event.transforms import (
+    SphereSpinToCartesianSpinTransform,
+    MassRatioToSymmetricMassRatioTransform,
+)
+
+
+def generate_fidiual_population(
```

Comment:
This is a function in sketch mode. It generates a sample of a fiducial population and save it to a .csv.

While the generate population part can remain here, I think saving to a csv shouldn't be here. Nonetheless, this doesn't hurt the original jim as I see it. Simply putting a TODO in will be enough